Contributing Member use case (#288)
ElektraInitiative/PermaplanT: doc/usecases/contributing_member.md

# Use Case: Contributing Member

## Summary

- **Scope:** Membership
- **Level:** User Goal
- **Actors:** App User
- **Brief:** The user receives a membership without compensation via money through contributions.
- **Status:** Draft

## Scenarios

- **Precondition:**
  - The user already has an existing membership OR
  - The user has currently no membership.
- **Main success scenario:**
  - The user can receive permacoins through various activities in the app, like:
    - gaining [Blossoms](./gamification/gain_blossoms.md) in the Expert Track.
    - getting likes from other users on their reviews.
    - getting [approvals](./polyculture/review_plant_relationships.md) on their submitted [plant relationships](./polyculture/add_plant_relationships.md).
    - getting their comments on a map marked as helpful by the maps owner.
    - recruiting new members.
  - The user can see their exact count of current and previous permacoins in their profile.
  - If the user applied for the contributing membership and when reaching a certain milestone with the amount of gathered permacoins, the user will be given the current calendar year of membership without compensation via money.

>   - Permacoins can not be transferred from the current into the next year.

Reviewer: ```suggestion
  - Permacoins can not be transferred from the current into another year nor be exchanged to money in any way.
```

- **Alternative scenario:**
  - The user does not gather enough permacoins and has to pay the full price for this years membership.
- **Error scenario:**
- **Postcondition:**

>   - The user can use the app without compensation via money if (s)he is a valuable member of the community.

Reviewer: ```suggestion
  - The user has the permission to use the app and to be part of the community.
```
